src/jvm/test/resources/expected_logs/run_concurrent_test/custom_threads_trace_debugger.txt

```diff
@@ -15,7 +15,7 @@ The following interleaving leads to the error:
 |     ArrayList#1.add(Thread#1): true                    |                                                                                                             |                                                                                                             |                                                                                                             |
 |     ArrayList#1.add(Thread#2): true                    |                                                                                                             |                                                                                                             |                                                                                                             |
 |     ArrayList#1.add(Thread#3): true                    |                                                                                                             |                                                                                                             |                                                                                                             |
-|     threads.iterator(): Itr#1                          |                                                                                                             |                                                                                                             |                                                                                                             |
+|     iterator(): Itr#1                                  |                                                                                                             |                                                                                                             |                                                                                                             |
```

Comment:
It should remain `threads.iterator()`

Reply:
How did it work before the change?

Reply:
I see, thank you for the clarification. What would we ideally see in the trace then?